prow/plugins/trigger/pull-request.go

```diff
@@ -132,7 +132,11 @@ func buildAllIfTrusted(c Client, trigger *plugins.Trigger, pr github.PullRequest
 			}
 		}
 		c.Logger.Info("Starting all jobs for updated PR.")
-		return buildAll(c, &pr.PullRequest, pr.GUID, trigger.ElideSkippedContexts)
+		elide := false
```

Comment:
might make sense to pull a default value from where the config is specified, otherwise the default is defacto in-line here.